sigs.yaml

```diff
@@ -717,6 +717,15 @@ sigs:
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/alibaba-cloud-csi-driver/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes/cloud-provider-alibaba-cloud/master/OWNERS
+    meetings:
+    - description: Regular Alibaba Cloud Subproject Meeting
+      day: Tuesday
+      time: "12:00"
+      tz: UTC
+      frequency: 'monthly 2020 start date: Jan. 7th'
+      url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
```

Comment:
I see this link also used by the other SIGs, maybe you can create your own?

Reply:
This is not edited by me. I just edit sigs.yaml. This line was added automatically by `make`.

Reply:
This link was used by other cloud providers, I think maybe this is common tradition. Is there any recommended url？

Reply:
The contents of that doc suggest that you should get a WG level doc soon. I think its fine for now but we should probably fix this once you get that doc.